Library/.rubocop.yml

```diff
@@ -7,124 +7,126 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+# enable all formulae audits
 FormulaAudit:
   Enabled: true
 
+# enable all formulae strict audits
 FormulaAuditStrict:
   Enabled: true
 
+# disable all formulae strict audits by default
 NewFormulaAudit:
   Enabled: false
 
 # `system` is a special case and aligns on second argument
 Layout/AlignParameters:
   Enabled: false
 
+# this is a bit less "floaty"
 Layout/CaseIndentation:
   EnforcedStyle: end
 
-Layout/EmptyLineBetweenDefs:
-  AllowAdjacentOneLineDefs: true
-
+# this is a bit less "floaty"
 Layout/EndAlignment:
-  EnforcedStyleAlignWith: variable
-
-Layout/IndentArray:
-  EnforcedStyle: special_inside_parentheses
+  EnforcedStyleAlignWith: start_of_line
 
+# enforce use of <<~EOS
 Layout/IndentHeredoc:
   EnforcedStyle: squiggly
 
 # conflicts with DSL-style path concatenation with `/`
 Layout/SpaceAroundOperators:
   Enabled: false
 
-# favor parens-less DSL-style arguments
+# # favor parens-less DSL-style arguments
```

Comment:
Is the second `#` a typo?

(Also favor? 🇺🇸 👀, heh)

Reply:
Yes, fixing (both 😉)

Reply:
There's a few of both here but I won't point them all out to avoid you wanting to prod me with a stick, justifiably 😄